include/zephyr/dt-bindings/pcie/pcie.h

```diff
@@ -35,6 +35,16 @@
 
 #define PCIE_ID_NONE PCIE_ID(0xFFFF, 0xFFFF)
 
+/* Helper macro to exclude invalid PCIe identifiers. We should really only
+ * need to look for PCIE_ID_NONE, but because of some broken PCI controllers
+ * we have try to cases where both VID & DID are zero or just one of them is
```

Comment:
the sentence is rather complex for non English speakers ;)

Reply:
The biggest problem was that it was in the wrong header file (dt-bindings instead of drivers). That's now fixed. I also clarified the documentation a little bit, but not sure how to make it any more clear than that.